src/forms/CalendarComponent.jsx

```diff
@@ -38,25 +36,30 @@ class CalendarComponent extends React.Component {
 			defaultDate: this.props.value
 		};
 
-		Object.assign(options, this.props.opts);
+		Object.assign(options, this.props.datepickerOptions);
```

Comment:
I see that it's existing code, but there's generally no need for `Options.assign` when you can use object destructuring - this is the same as adding `...this.props.datepickerOptions` as the last line of the `options` definition.